cnf/ext/junit.bnd

```diff
@@ -1,6 +1,20 @@
 junit: org.apache.servicemix.bundles.junit;version="[4.11,5)",\
- org.assertj.core;version="[3,4)"
+	org.assertj.core;version="[3,4)"
+junit-platform-version: 1.5.2
```

Comment:
Not sure I like these version numbers in here. I think they should just be directly specified in central.mvn.

Reply:
Your call. There are several modules in each group of dependencies that need to be kept at the same version, I thought using a macro for DRY was appropriate.